### PR TITLE
Change webview link clicks open in device browser (android) (closes #105)

### LINKED ISSUE #105: Links inside webview should open in browser

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@samgiles r?

File: android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewClient.java [hunk 1]
```diff
@@ -22,24 +22,38 @@ public void onPageFinished(WebView webView, String url) {
         Handler handler = new Handler();
         handler.postDelayed(new Runnable() {
             public void run() {
+                magnetWebView.loaded = true;
```
Comment (wilsonpage) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewClient.java]: Is this a hacky pattern? #androidnoob

Comment (samgiles) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewClient.java]: Yeh it is a bit, it's kind of weird that the client sets the loaded property.

Comment (wilsonpage) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewClient.java]: What works better?

Comment (samgiles) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewClient.java]: Seems like just storing the state in the client makes sense.

File: android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
 public class MagnetWebView extends WebView implements LifecycleEventListener {
     String TAG = "MagnetWebView";
     private boolean layoutSet;
+    public boolean loaded = false;
```
Comment (samgiles) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java]: I think adding a public property, that's not used by the owner is a hint that something's up.  See comment below.

File: android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java [hunk 3]
```diff
@@ -15,6 +15,7 @@
 public class MagnetWebView extends WebView implements LifecycleEventListener {
     String TAG = "MagnetWebView";
```
Comment (samgiles) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebView.java]: Let's take the opportunity to make this `private static final String TAG = "MagnetWebView"`.

**Comment by samgiles** [comment]
With comments addressed LGTM!

**Comment by wilsonpage** [comment]
@samgiles comments addressed. Landing on green.

**Pull request merged**